src/vorta/notifications.py

```diff
@@ -78,7 +78,7 @@ def _dbus_notify(self, header, msg, level='info'):
         v = QtCore.QVariant(12321)  # random int to identify all notifications
         if v.convert(QtCore.QVariant.UInt):
             id_replace = v
-        icon = ""
+        icon = "com.borgbase.Vorta-symbolic"
```

Comment:
I tested this as "icon = 'com.borgbase.Vorta'" the notification showed the icon. Is there a need for a seperate icon?

Reply:
I wondered the same.
Our icon is already very simplistic.
But normally symbolic icons are only black-white